main.js

```diff
@@ -7,6 +7,14 @@ function loadConnections() {
     $connectionContainer.empty();
   }
 
+  console.log(connections)
+  if(Object.keys(connections).length === 0) {
+    // no connection, add default
+    console.log("Adding default connection")
+    storageService.connections.save({url: "broker.mqttdashboard.com", port: "8000", username: "", password: ""})
```

Comment:
Would it make sense to put the default connection object in a constant at the top of the main.js?